### PR TITLE
Build docker images

### PR DESCRIPTION
**Pull request by szok**
The build scripts for Ralph Docker image

### CODE REVIEW

File: os/upgrade_mysql.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (mkurek) [inline os/upgrade_mysql.sh]: don't you think that `upgrade_ralph` filename will suit better than `upgrade_mysql`?

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+if [ "$instance" == "os" ]; then 
+	image_name="vi4m/ralph:latest"
+fi
+	
+if [ "$1" = "build" ]; then
+	./$instance/build.sh 
+fi
+
+if [ "$1" = "debug" ]; then
+	sudo docker run  --rm=true --name ${instance} -p 9000:9000 -t -i --volumes-from ${instance}_storage devdemo $3 $4 $5 $6 $7 $8
```
Comment (mkurek) [inline run.sh]: devdemo?

File: os/upgrade_mysql.sh (hunk 1 above)
Comment (szok) [inline os/upgrade_mysql.sh]: Fact :)

File: base/build.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+instance="base"
+dir="/home/ubuntu/docker-configs/base"
```
Comment (mkurek) [inline base/build.sh]: definitely  not `/home/ubuntu/...`

try using sth like this to get current directory: `DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )`

File: os/build.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
+
+branch='develop'
+
+instance="os"
+dir="/home/ubuntu/docker-configs/os"
```
Comment (mkurek) [inline os/build.sh]: same situation as in base/build.sh

File: base/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Ralph
+#
+# VERSION               2.0.0-3
+ 
+FROM      ubuntu:12.04
+MAINTAINER Marcin Kliks marcin.kliks@allegro.pl
```
Comment (vi4m) [inline base/Dockerfile]: pylabs@allegrogroup.com

File: os/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# Ralph
+#
+# VERSION               2.0.0-3
```
Comment (vi4m) [inline os/Dockerfile]: remove it all

File: os/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+ 
+FROM      base
+MAINTAINER Marcin Kliks marcin.kliks@allegro.pl
```
Comment (vi4m) [inline os/Dockerfile]: pylabs@allegrogroup.com

**Comment by vi4m** [comment]
lgtm

**Comment by mkurek** [comment]
:ok:

File: README.md [hunk 8]
```diff
@@ -1 +1,38 @@
...
+# Ralph Docker image configuration
+
+Build docker container to Ralph application
+
+Build docker os image:
+```
+sudo ./run.sh build base
+```
+
+Build docker os image:
```
Comment (quamilek) [inline README.md]: The same title, please describe difference between `base` and `os` image.

File: os/build.sh [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+cd sources/
+git clone https://github.com/allegro/ralph.git -b $branch
+git clone https://github.com/allegro/ralph_assets.git -b $branch
+git clone https://github.com/allegro/ralph_pricing.git -b $branch
+git clone https://github.com/allegro/django-bob.git django_bob -b $branch
+git clone https://github.com/quamilek/bob-ajax-selects.git -b $branch
+cd ..
+cp Makefile sources/
+
+docker build -t='vi4m/ralph:latest' .
```
Comment (quamilek) [inline os/build.sh]: If this image will be official Ralph docker image, maybe we will use/create official AllegroGroup account.

File: os/createsuperuser.py [hunk 10]
```diff
@@ -0,0 +1,11 @@
+
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ralph.settings")
+
+from django.contrib.auth.models import User
+
+u = User.objects.get(username='ralph')
```
Comment (quamilek) [inline os/createsuperuser.py]: `u` -> `user` looks better.

File: os/build.sh (hunk 9 above)
Comment (vi4m) [inline os/build.sh]: ok. quamilek can you take care of this in the future?

**Pull request merged**